src/css/Book.css

```diff
@@ -1,46 +1,98 @@
 .BookCard {
-  width: 75rem;
-  height: 2.625rem;
-  margin: 2.313rem 6.25rem 0;

```

Comment:
- [OPTIONAL] Make sure all line bars are aligned in the same line as shown on the image below 👇🏻 

![18](https://user-images.githubusercontent.com/79058364/192021092-36bd74e7-542d-4173-aa3e-8927f5d4475c.png)